Java/core/src/main/java/com/amazon/randomcutforest/store/PointStore.java

```diff
@@ -744,6 +744,19 @@ public PointStore(PointStore.Builder builder) {
                         : new float[dimensions];
             }
 
+            /*
+             * an explicit conversion to bits; using toBits(refCount) in the constructor of
+             * IndexIntervalManager converts 256 into 0 and for a location appearing 256
```

Comment:
Can we fix this in the IndexIntervalManager instead?

Reply:
Done